src/IonParserBinaryRaw.ts

```diff
@@ -494,6 +494,7 @@ export class ParserBinaryRaw {
             a = (a << VINT_SHIFT) | (b & VINT_MASK);  // OR in the 7 useful bits
             if ((b & VINT_FLAG) !== 0) {
                 // once we have the last byte, add it to our list and start the next
+                if(a === 0) throw new Error('Symbol ID zero is unsupported.');
```

Comment:
Matter of preference: I like having the contents of an `if` statement wrapped in braces and on the following line(s), even if it's only a single statement. If WebStorm's code formatter doesn't mind though, I'm happy to leave it as is.